views/layouts/layout.ejs

```diff
@@ -1,13 +1,20 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="/styles/lvlOfKnowledge.css">
```

Comment:
I would also suggest adding https://necolas.github.io/normalize.css/ to your project. Simply copy the contents into a `.css` file and include it in the head _before_ your custom CSS file. 

Read more about it [here](http://nicolasgallagher.com/about-normalize-css/) if you're curious.